WalletWasabi.Fluent/Behaviors/TextBoxInputLimitBehavior.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Input.Platform;
+using Avalonia.Interactivity;
+using Avalonia.Threading;
+using ReactiveUI;
+
+namespace WalletWasabi.Fluent.Behaviors
+{
+	public class TextBoxInputLimitBehavior : DisposingBehavior<TextBox>
+	{
+		private int _pasteCharCountLimit;
+
+		public static readonly DirectProperty<TextBoxInputLimitBehavior, int> PasteCharCountLimitProperty =
+			AvaloniaProperty.RegisterDirect<TextBoxInputLimitBehavior, int>(nameof(PasteCharCountLimit),
+				o => o.PasteCharCountLimit,
+				(o,
+					v) => o.PasteCharCountLimit = v, unsetValue: 120);
```

Comment:
@danwalmsley is 120 chars a good limit for this?

Reply:
150 is the minimum for passwords, so IMO we should go with 150

Reply:
okay :+1: